### ISSUE TITLE
Issues with next version of ggplot2

### ISSUE THREAD

**Issue by thomasp85** [davidchall/ggip#24]
Hi

We preparing the next release of ggplot2 and our reverse dependency checks show that your package is failing with the new version. Looking into it we see that this is due to changes in the warnings and errors thrown by ggplot2 that you test for in your package

You can install the release candidate of ggplot2 using `devtools::install_github('tidyverse/ggplot2@v3.4.0-rc')` to test this out.

We plan to submit ggplot2 by the end of October and hope you can have a fix ready before then

Kind regards
Thomas

**Comment by davidchall** [comment]
Hi @thomasp85, thanks for the advance warning!

I recently made a patch release for compatibility with changes to {tidyselect}, so I think I need to wait until Oct 29 for the next patch release. It sounds like this will work with your schedule though.

**Comment by davidchall** [comment]
~Unfortunately, I think I'll need to archive ggip because these test failures are substantial. The ggip package was always using ggplot2 in a rather strange way (mapping a single data variable to a 2D position), and it looks like it's time has finally run out.~ 😄 

_Update:_ I discovered the errors were caused by https://github.com/tidyverse/ggplot2/issues/5019. If the latest vctrs package is available, then I can get ggip working with ggplot2 3.4.0.
